extension-ui/content.js

```diff
@@ -1,18 +1,29 @@
 let blockedSitesCache = null;
```

Comment:
remove these changes from your branch

Reply:
I have no idea what were the changes that were done in the branch

Reply:
please copy the original content from main and replace